src/main/java/com/softserveinc/Task01.java

```diff
@@ -16,9 +15,42 @@
  */
 public class Task01 {
     public static void main(String[] args) {
-                              //  I  | II | III | IV |  V  |  VI | VII | VIII | IX  |  X  | XI |  XII
-        double[] temperature = { -1.7, -1.0, 2.6,  9.0, 15.1, 19.4, 21.4, 21.2, 17.1, 11.1, 5.9,  1.4 };
-        // TODO: Write your code here
+        //  I  | II | III | IV |  V  |  VI | VII | VIII | IX  |  X  | XI |  XII
+        double[] temperature = {-1.7, -1.0, 2.6, 9.0, 15.1, 19.4, 21.4, 21.2, 17.1, 11.1, 5.9, 1.4};
+        String[] rome = {"I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX", "X", "XI", "XII"};
+        double sum = 0;
+        double min = temperature[0];
+        double max = temperature[0];
+
+        for (double i : temperature) {
+            sum = sum + i;
+
+            if (min > i) {
```

Comment:
You could use loop `for` with counter and it will give you the possibility to figure out the index the warmest and the coldest month.